code/game/turfs/open/floor.dm

```diff
@@ -44,6 +50,11 @@
 	if(is_station_level(z))
 		GLOB.station_turfs += src
 
+/turf/open/floor/proc/setup_broken_states()
+	broken_states = string_list(list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5"))
```

Comment:
```suggestion
	broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5")
```
This will already be turned into a stringlist on initialization.